src/career.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Career</title>
+  </head>
+  <body>
+    <h1>The Career of Keanu Reeves</h1>
+    <!-- Intro with basic info on Reeves -->
+    <h2>Intro to Keanu Reeves</h2>
+    <p>
+      Keanu Reeves is a Canadian Movie star that is famous for his many
+      blockbuster roles. Staring in the Bill and Ted franchise, the Matrix
+      franchise and the John Wick movies. To name a few. Reeves also plays the
+      bass guitar for the bands Dogstar and Becky. He has also Acted onstage, he
+      performed as Prince Hamlet for the Manitoba Theatre Centre's production of
+      Hamlet. He has also directed one movie Man of Chi.
+      <!-- List of Skill and talents -->
+    </p>
+
+    <h2>Skills and Talents</h2>
+    <ul>
+      <li>Acting of course</li>
+      <li>Playing base guitar</li>
```

Comment:
*bass

Reply:
Those should now be fixed with the new version of this PR. Thank you for the feedback.